packages/preview/badgery/0.1.0/typst.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "badgery"
+version = "0.1.0"
+entrypoint = "badgery.typ"
+authors = ["dogezen"]
+license = "MIT"
+description = "Adds styled badges, boxes and menu actions."
+compiler = "0.11.0"
+keywords = ["badge", "boxes"]
+repository = "https://github.com/dogezen/badgery"
```

Comment:
You might want to add [`categories`](https://github.com/typst/packages/blob/main/CATEGORIES.md) (and possibly also [`disciplines`](https://github.com/typst/packages/blob/main/DISCIPLINES.md) if you think it makes sense). This will make the package more discoverable on [Typst Universe](https://typst.app/universe).

Reply:
Good suggestion - have now added a category and some disciplines.